yang/frr-pim.yang

```diff
@@ -56,11 +56,11 @@ module frr-pim {
      (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
      OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.";
 
-  revision 2017-03-09 {
+  revision 2021-11-22 {
     description
       "Initial revision.";
     reference
-      "RFC XXXX: A YANG Data Model for PIM";
+      "RFC 7761: A YANG Data Model for PIM";
```

Comment:
YANG data model for PIM is still a draft: https://datatracker.ietf.org/doc/draft-ietf-pim-yang/
Why this change?

Reply:
Removed the change